app/views/homepages/index.html.erb

```diff
@@ -0,0 +1,24 @@
+
+<section class="spotlight">
+<% media = Work.all.sample%>
+    <h2 class="spotlight__header">
+        <span class="spotlight__header--prefix"> Media Spotlight </span>
```

Comment:
This should probably use the `@spotlight` variable that you set up in the controller. As-is it shows a random work, not the one with the most votes.